tests/integration/conftest.py

```diff
@@ -289,7 +289,7 @@ async def app_no_wait_fixture(
         app_name=app_name,
         path=path,
         token=token,
-        runner_storage="memory",
+        runner_storage="juju-storage",
```

Comment:
Why the change here? @yanksyoon

Reply:
@cbartz 
There seems to be a problem with the config option `memory`, see this [workflow run](https://github.com/canonical/github-runner-operator/actions/runs/8043111866/job/21964781492#step:13:191).
I'm trying to fix the `StopIteration` error first by using `tzutc` and having a bit of padding in start time when querying for workflow runs.
This should be tackled once the `StopIteration` issue is fixed :D